partiql-lang-kotlin-omitted-tests.md

```diff
@@ -0,0 +1,70 @@
+The following lists out the tests from `partiql-lang-kotlin`'s parse tests that are not part of `partiql-tests`:
+
+Tests excluded:
+- Custom type cast
+  - https://github.com/partiql/partiql-lang-kotlin/blob/34625c68dbcbaf7b8ae60df7a4cf65c60b2a3b79/lang/test/org/partiql/lang/syntax/SqlParserTest.kt#L631-L644
+  - [SqlParserCastTests.kt](https://github.com/partiql/partiql-lang-kotlin/blob/34625c68dbcbaf7b8ae60df7a4cf65c60b2a3b79/lang/test/org/partiql/lang/syntax/SqlParserCastTests.kt)
+  - [SqlParserCustomTypeCatalogTests.kt](https://github.com/partiql/partiql-lang-kotlin/blob/34625c68dbcbaf7b8ae60df7a4cf65c60b2a3b79/lang/test/org/partiql/lang/syntax/SqlParserCustomTypeCatalogTests.kt) 
+- ORDER BY tests testing defaults for sort spec and null spec
+  - https://github.com/partiql/partiql-lang-kotlin/blob/34625c68dbcbaf7b8ae60df7a4cf65c60b2a3b79/lang/test/org/partiql/lang/syntax/SqlParserTest.kt#L1793-L1888
+  - dependent on when parsed ast checking is added
```

Comment:
This could be tested via correspondence without needing to test the AST.

e.g. 
```
let defaulted = parse("SELECT x FROM tb ORDER BY rk1, rk2, rk3, rk4");
let explicit = parse("SELECT x FROM tb ORDER BY rk1 NULLS LAST, rk2 NULLS LAST, rk3 NULLS LAST, rk4 NULLS LAST");
assert_eq!( canonicalize(defaulted), canonicalize(explicit) );
```

Though, this is perhaps also better tested via evaluator....